<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wine Prefix Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>My Apps and Games</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Add New App or Game</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-add-new-100.png</normaloff>:/newPrefix/icons/icons8-add-new-100.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>Run Windows File</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-powershell-48.png</normaloff>:/newPrefix/icons/icons8-powershell-48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>661</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Select to Run</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Wine Prefix Manager</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>Registery Editor</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-registry-editor-96.png</normaloff>:/newPrefix/icons/icons8-registry-editor-96.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>Uninstaller</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-uninstalling-updates-64.png</normaloff>:/newPrefix/icons/icons8-uninstalling-updates-64.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Control Panel</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-control-panel-80.png</normaloff>:/newPrefix/icons/icons8-control-panel-80.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>CMD</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-console-64.png</normaloff>:/newPrefix/icons/icons8-console-64.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Kill All Processes</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-remove-96.png</normaloff>:/newPrefix/icons/icons8-remove-96.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Task Manager</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-system-task-64.png</normaloff>:/newPrefix/icons/icons8-system-task-64.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Winecfg</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-config-58.png</normaloff>:/newPrefix/icons/icons8-config-58.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Restart</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-restart-58.png</normaloff>:/newPrefix/icons/icons8-restart-58.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>File Explorer</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-file-explorer-64.png</normaloff>:/newPrefix/icons/icons8-file-explorer-64.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string>Internet Explorer</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/newPrefix/icons/icons8-internet-explorer-64.png</normaloff>:/newPrefix/icons/icons8-internet-explorer-64.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>dependencies</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>   Install or Update Vulkan</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/newPrefix/icons/icons8-3d-64.png</normaloff>:/newPrefix/icons/icons8-3d-64.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>661</width>
        <height>321</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>659</width>
         <height>319</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Add to Registery</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>641</width>
         <height>89</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Key : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Data : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Value : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/newPrefix/icons/icons8-add-new-100.png</normaloff>:/newPrefix/icons/icons8-add-new-100.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>661</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Windows Version :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Windows 10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Windows 8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Windows 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Windows Vista</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Windows XP</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>661</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Virtual Desktop :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>On</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Off</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3">
         <item>
          <property name="text">
           <string>800x600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1920x1080</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>661</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Set Dpi : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>90</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>120</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
